README.md

```diff
@@ -82,16 +88,17 @@ $ mn setup testnet masternode
 The `config` command is used to manage your node configuration before starting the node. Several system configurations are provided as a starting point:
 
  - base - basic config for use as template
- - local - standalone node for local development
- - testnet - node with testnet configuration
+ - local - template for local node configs
```

Comment:
```suggestion
 - local - template for local node configurations
```

Reply:
I guess we used "configs" intentionally to point to config entities. Does it make sense to change it to "configurations" anyways?

Reply:
No worries. It's fine to leave as configs